packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -2414,6 +2418,74 @@ void main() {
 
     expect(find.byType(CupertinoPickerDefaultSelectionOverlay), findsExactly(4));
   });
+
+  testWidgets('CupertinoDatePicker month column width validation in Portuguese', (WidgetTester tester) async {
+    // In Portuguese, the longest month name is 'Fevereiro' (February).
+    // But the widest month name is 'Novembro' (November).
+    const String largestMonthText = 'Novembro';
+
+    await _loadFonts();
+
+    CupertinoThemeData theme = const CupertinoThemeData();
+    theme = theme.copyWith(
+        textTheme: theme.textTheme.copyWith(
+            dateTimePickerTextStyle: theme.textTheme.dateTimePickerTextStyle
+                .copyWith(fontFamily: 'Roboto')));
+
+    await tester.pumpWidget(CupertinoApp(
+      theme: theme,
+      locale: const Locale('pt'),
+      supportedLocales: const <Locale>[Locale('pt')],
+      localizationsDelegates: const <LocalizationsDelegate<
+          CupertinoLocalizations>>[_MockDefaultCupertinoLocalizations.delegate],
+      home: Builder(builder: (BuildContext context) {
+        return Center(
+          child: CupertinoDatePicker(
+              mode: CupertinoDatePickerMode.monthYear,
+              onDateTimeChanged: (DateTime date) {},
+              initialDateTime: DateTime(2018, 11, 15)),
+        );
+      }),
+    ));
+
+    // pump twice required for loading localization
```

Comment:
I'm curious as to why it takes 2 frames to load the localization?

Reply:
This occurred because I was taking the measurements in each case and not at the end.